serverless.yml

```diff
@@ -17,9 +17,9 @@ provider:
   runtime: nodejs10.x
   stackName: ${self:service.name}-stack
   stage: ${opt:stage, 'prod'}
-  region: ${opt:region, 'eu-central-1'}
+  region: eu-central-1
   deploymentBucket:
-    name: ${env:SLS_DEPLOY_BUCKET, 'kaskadi-serverless-deployment-bucket'}
+    name: 'kaskadi-serverless-deployment-bucket'
   deploymentPrefix: ${self:service.name}
```

Comment:
what is deploymentPrefix used for?

Reply:
To be able to choose a prefix for the deployment. For example if I use `@kaskadi/my-resource` it would upload the files inside of our bucket under the path `@kaskadi/my-resource` (it's creating the folder structure itself)

Reply:
Otherwise it's defaulting to a generic name built from the config file so I'm using it to ensure we are consistent + if I want to use a common folder (for example `@kaskadi`) it's allowing me to do so

Reply:
but thats just for the deploy bucket, right? no one is ever going to look into that bucket anyways... no need for consistency as long as we are sure there will be no naming collisions...

Reply:
Yep only for the deployment. You think we just get rid of it and upload everything as flat structure with default name then?

Reply:
how are the default names constructed?

Reply:
Looks like it's pulling from service.name so the same way as I am using it

Reply:
minus the folder structure flexibility that we don't care about then

Reply:
then i'd say remove `deploymentPrefix`